.github/workflows/validate_issue.yml

```diff
@@ -1,7 +1,7 @@
 name: Validate issue
 on:
   issues:
-    types: [opened, labeled]
+    types: [labeled]
```

Comment:
This triggered the action twice for new issues, because "labeled" already gets triggered for the "template: bug" default label.